Setup.sh

```diff
@@ -94,27 +94,27 @@ markEndOfBlock "Create folders"
 markStartOfBlock "Retrieve dependencies"
```

Comment:
Thanks for updating this file also - @jessicafalk thanks you :)

Reply:
oh yes, you have no idea how happy this makes me :D

Reply:
I must say this file is untested, though.